src/main/webapp/app/shared/metis/post/post.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="row module-bg mb-1 pb-1 rounded-2" [ngClass]="{ 'mx-0': !isThreadSidebar }">
+<div class="row module-bg mb-1 pb-1 rounded-2" [ngClass]="{ 'mx-0': !isThreadSidebar, 'highlight-pinned': posting.displayPriority === DisplayPriority.PINNED }">
```

Comment:
I think this would be a bit cleaner:
```suggestion
<div class="row module-bg mb-1 pb-1 rounded-2" [class.mx-0]="!isThreadSidebar" [class.highlight-pinned]="posting.displayPriority === DisplayPriority.PINNED">
```